### PR TITLE
Formatchange

### PR DESCRIPTION
**Pull request by MarkusHackspacher**
Change to the .format() function.

I make not changes at:
https://github.com/unknown-horizons/unknown-horizons/blob/master/horizons/gui/widgets/messagewidget.py#L351-L356
I think the unicode is not need with python 3.

### CODE REVIEW

File: horizons/gui/modules/settings.py [hunk 1]
```diff
@@ -222,13 +222,13 @@ def slider_change(self, widget):
...
 		value = {
-			'volume_music':      lambda x: '%s%%' % int(500 * x),
-			'volume_effects':    lambda x: '%s%%' % int(200 * x),
-			'mousesensitivity':  lambda x: '%+.1f%%' % (200 * x),
-			'autosaveinterval':  lambda x: '%d' % x,
-			'autosavemaxcount':  lambda x: '%d' % x,
-			'quicksavemaxcount': lambda x: '%d' % x,
-			'scrollspeed':       lambda x: '%.1f' % x,
+			'volume_music':      lambda x: '{:d}%%'.format(int(500 * x)),
+			'volume_effects':    lambda x: '{:d}%%'.format(int(200 * x)),
```
Comment (AndyMender) [inline horizons/gui/modules/settings.py]: This line and the line above - you don't have to double the percentage '%' sign in `.format()` string templates:
```py
>>> "{:d}%".format(200)
'200%'
>>> "{:d}%%".format(200)
'200%%'
>>> 
```

File: horizons/gui/modules/settings.py [hunk 2]
```diff
@@ -222,13 +222,13 @@ def slider_change(self, widget):
...
+			'mousesensitivity':  lambda x: '{:+.1f}%%'.format(200 * x),
```
Comment (AndyMender) [inline horizons/gui/modules/settings.py]: Same as in the previous case.

File: horizons/gui/modules/settings.py [hunk 3]
```diff
@@ -222,9 +222,9 @@ def slider_change(self, widget):
 		a '%' suffix."""
 		value_label = self.widget.findChild(name=widget.name + '_value')
 		value = {
-			'volume_music':      lambda x: '{:d}%%'.format(int(500 * x)),
-			'volume_effects':    lambda x: '{:d}%%'.format(int(200 * x)),
-			'mousesensitivity':  lambda x: '{:+.1f}%%'.format(200 * x),
+			'volume_music':      lambda x: '{:d}%'.format(int(500 * x)),
+			'volume_effects':    lambda x: '{:d}%'.format(int(200 * x)),
+			'mousesensitivity':  lambda x: '{:+.1f}%'.format(200 * x),
```
Comment (AndyMender) [inline horizons/gui/modules/settings.py]: These changes are okay :).

File: horizons/gui/modules/settings.py (hunk 2 above)
Comment (AndyMender) [inline horizons/gui/modules/settings.py]: For the lines above - you don't have to double the percentage '%' sign in `.format() `string templates:
```py
>>> "{:d}%".format(200)
'200%'
>>> "{:d}%%".format(200)
'200%%'
>>> 
```

Comment (MarkusHackspacher) [inline horizons/gui/modules/settings.py]: That are the same lines, the first change was from ... '%d%%' % x to ... '{:d}%%'.format( and the second was right ...'{:d}%'.format(

Sorry for the confusion.

Comment (AndyMender) [inline horizons/gui/modules/settings.py]: Ah, my apologies then! I got confused with the commit order.

**Comment by MarkusHackspacher** [comment]
From 110 issues in 42 files to 78 issues in 32 files.

**Pull request merged**